ui-modules/app-inspector/app/components/entity-effector/modal/modal.template.html

```diff
@@ -59,8 +59,37 @@ <h3>Parameters</h3>
             </div>
         </div>
     </section>
+    <br ng-if="vm.cli"/>
+    <section ng-if="vm.cli">
+        <h3>CLI</h3>
+        <div class="cli-content">
+            <span class="extras">
+                <button class="btn copy-btn btn-default btn-sm" ngclipboard ngclipboard-success="onClipboardSuccess(e)"
+                        data-clipboard-text="{{brCmd()}}" uib-tooltip="Copied!"
+                        tooltip-trigger="{'copied': 'mouseleave'}" tooltip-placement="left"
+                        tooltip-append-to-body="true">
+                    <i class="fa fa-clipboard"></i>
+                </button>
+            </span>
+            <pre>{{brCmd()}}</pre>
+        </div>
+    </section>
+    <section ng-if="vm.cli">
+        <h3>REST</h3>
```

Comment:
Think `API` could it more clear for non-dev Brooklyn operators

Reply:
I'll keep `REST`, as it was agreed earlier.

Reply:
Changing it to API as there are 2 votes for it.